### PR TITLE
Api 2523:  Hooks migration for Home and News

### PR DESCRIPTION
**Pull request by TS000**
### Description
https://vajira.max.gov/browse/API-2523
Moved Class and Function based React components to modern Hooks. 

### Process
Process was simple as the components were pretty shallow in their functionality. 

### Requested feedback
This is my first commit, please let me know if I need to adjust how I'm pushing commits. I'm also waiting on my proxy access to allow me to look at Jenkins issues.

### CODE REVIEW

File: src/containers/Home.tsx [hunk 1]
```diff
@@ -14,103 +14,93 @@ const rightColumnClasses = classNames('medium-screen:vads-l-col--8', 'small-scre
...
-                      name={`VA ${name}`}
-                      url={`/explore/${apiCategoryKey}`}
-                    >
-                      {content.placardText}
-                    </CardLink>
-                  </Flag>
-                );
-              })}
-            </div>
+const ApiList = () => (
```
Comment (alexandervalencia) [inline src/containers/Home.tsx:35]: See `JSX.Element` comment above

File: src/containers/Home.tsx [hunk 2]
```diff
@@ -14,103 +14,93 @@ const rightColumnClasses = classNames('medium-screen:vads-l-col--8', 'small-scre
...
-  return (
-    <section className="vads-u-display--none medium-screen:vads-u-display--block" role="region" aria-label="Veterans Notice">
-      <div className={classNames(
-        'vads-u-background-color--primary',
-        'vads-u-font-size--lg',
-        'vads-u-padding-y--0p5',
-        'vads-u-text-align--center',
-        'vads-u-color--white')}>
-        <div className='vads-u-margin-y--2p5'>
+const VeteransNotice = () => (
```
Comment (alexandervalencia) [inline src/containers/Home.tsx:63]: See `JSX.Element` comment above

File: src/containers/Home.tsx [hunk 3]
```diff
@@ -14,103 +14,93 @@ const rightColumnClasses = classNames('medium-screen:vads-l-col--8', 'small-scre
...
 
-class Home extends React.Component {
-  public render() {
-    return (
-      <div className="home">
-        <Hero />
-        <VeteransNotice />
-        <HomeSection ariaLabel="An API platform" imageSrc={padlockImg} title="A secure API platform to service Veterans.">
-          <div>
+const Home = () => (
```
Comment (alexandervalencia) [inline src/containers/Home.tsx:79]: See `JSX.Element` comment above

File: src/containers/News.tsx [hunk 4]
```diff
@@ -34,11 +34,10 @@ const sections = NewsData.sections.map((section: DataSection) => ({
   id: toHtmlId(section.title),
 }));
 
-function NewsItem({ item, media }: { item: NewsItem; media: boolean }) {
-  return media ? <MediaItem item={item} /> : <ItemDescription item={item} />;
-}
+const NewsItem = ({ item, media }: { item: NewsItem; media: boolean }) =>
```
Comment (alexandervalencia) [inline src/containers/News.tsx:37]: See `JSX.Element` comment above

File: src/containers/News.tsx [hunk 5]
```diff
@@ -34,11 +34,10 @@ const sections = NewsData.sections.map((section: DataSection) => ({
...
+  media ? <MediaItem item={item} /> : <ItemDescription item={item} />;
 
-function MediaItem({ item }: { item: NewsItem }) {
+const MediaItem = ({ item }: { item: NewsItem }) => {
```
Comment (alexandervalencia) [inline src/containers/News.tsx:40]: See `JSX.Element` comment above

File: src/containers/News.tsx [hunk 6]
```diff
@@ -62,22 +61,20 @@ function MediaItem({ item }: { item: NewsItem }) {
...
-      <a href={item.url}>{item.title}</a>
-      <br />
-      <strong>
-        {item.date}
-        {item.source ? ` | ${item.source}` : null}
-      </strong>
-    </p>
-  );
-}
+const ItemDescription = ({ item }: { item: NewsItem }) => (
```
Comment (alexandervalencia) [inline src/containers/News.tsx:66]: See `JSX.Element` comment above

File: src/containers/News.tsx [hunk 7]
```diff
@@ -62,22 +61,20 @@ function MediaItem({ item }: { item: NewsItem }) {
...
+    <br />
+    <strong>
+      {item.date}
+      {item.source ? ` | ${item.source}` : null}
+    </strong>
+  </p>
+);
 
-export default function News() {
+const News = () => {
```
Comment (alexandervalencia) [inline src/containers/News.tsx:77]: See `JSX.Element` comment above

**Review by anthony-kopczyk — APPROVED** [review]
Everything looks good to go!

**Review by alexandervalencia — COMMENTED** [review]
@TS000 Based on @johanna-adhoc 's PR https://github.com/department-of-veterans-affairs/developer-portal/pull/445 I made one small suggestion (applied a few places throughout the code)

**Review by alexandervalencia — DISMISSED** [review]
Looks good!

**Review by johanna-adhoc — APPROVED** [review]
💯

**Pull request merged**